packages/graphql-contentful-helpers/src/createSanityLoaders.ts

```diff
@@ -0,0 +1,91 @@
+import DataLoader from 'dataloader';
+import sanityClient, { SanityClient } from '@sanity/client';
+import { ContentfulLoaders, ItemKey, FVLKey, RefByKey } from '@last-rev/types';
+import LastRevAppConfig from '@last-rev/app-config';
+
+const createSanityClient = (config: any, useCdn: boolean): SanityClient => {
+  return sanityClient({
+    projectId: config.projectId,
+    dataset: config.dataset,
+    apiVersion: config.apiVersion || '2021-03-25',
+    token: config.token,
+    useCdn
+  });
+};
+
+const createSanityLoaders = (
+  config: LastRevAppConfig,
+  _defaultLocale: string
+): ContentfulLoaders => {
+  const sanityCfg = (config as any).sanity || {};
+  const prodClient = createSanityClient(sanityCfg, true);
+  const previewClient = createSanityClient(sanityCfg, false);
+
+  const getClient = (preview: boolean) => (preview ? previewClient : prodClient);
+
+  const entryLoader = new DataLoader<ItemKey, any | null>(async (keys) => {
+    const results = await Promise.all(
+      keys.map(({ id, preview }) =>
+        getClient(preview).fetch('*[_id == $id][0]', { id })
+      )
+    );
```

Comment:
We're not leveraging batching of ids for queries, this can hit rate limits in Sanity